internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceMultiClusterWrapper.java

```diff
@@ -129,6 +130,11 @@ static ServiceProvider<VeniceMultiClusterWrapper> generateService(VeniceMultiClu
       extraProperties.put(SYSTEM_SCHEMA_CLUSTER_NAME, clusterNames[0]);
       extraProperties.putAll(KafkaTestUtils.getLocalCommonKafkaSSLConfig(SslUtils.getTlsConfiguration()));
       pubBrokerDetails.forEach((key, value) -> extraProperties.putIfAbsent(key, value));
+      if (controllerProperties.containsKey(PARTICIPANT_MESSAGE_STORE_ENABLED)) {
+        extraProperties
```

Comment:
I think we should make this by default enabled for all the test. This feature is always turned on and our test should keep exercising the code path.

Reply:
The problem is if we turn it on lot of tests start failing. Unless we make them pass always we can turn it on

Reply:
I see..this is a bit concerning. But maybe we can improve that incrementally.

Reply:
I previously attempted to enable it in this [PR](https://github.com/linkedin/venice/pull/1338) but had to disable it again due to failing tests. However, I recall keeping it enabled for most tests while disabling it for specific ones. I’ll re-enable it in a follow-up PR, and if all tests pass, we can enable it by default.